probe-rs/src/bin/probe-rs/util/logging.rs

```diff
@@ -96,7 +96,7 @@ pub fn setup_logging(
     default: Option<LevelFilter>,
 ) -> anyhow::Result<Option<FileLoggerGuard<'_>>> {
     let stdout_subscriber = tracing_subscriber::fmt::layer()
-        .compact()
+        .pretty()
```

Comment:
Can we do this separately? I wouldn't mind going through a few lines less.

Reply:
I'll clean it up when I get this ready to merge.